src/sender.js

```diff
@@ -76,17 +76,22 @@ async function main({ msg, ackMsg, nackMsg, sendToQueue, channel }) {
     const gasPrice = await GasPrice.getPrice()
 
     const ttl = REDIS_LOCK_TTL * txArray.length
+
+    logger.debug('Acquiring lock')
     const lock = await redlock.lock(nonceLock, ttl)
 
+    logger.debug('Reading nonce')
     let nonce = await readNonce()
     let insufficientFunds = false
     let minimumBalance = null
     const failedTx = []
 
+    logger.debug(`Sending ${txArray.length} transactions`)
     await syncForEach(txArray, async job => {
       const gasLimit = addExtraGas(job.gasEstimate, EXTRA_GAS_PERCENTAGE)
 
       try {
+        logger.debug(`Sendint transaction with nonce ${nonce}`)
         const txHash = await sendTx({
```

Comment:
I think it is reasonable to have a logging for all transaction' raw data (as well as whole JSON sent to a RPC node). Consider to have an environment variable flag (e.g. `RPC_TRACES=true/false`) that will enable or disable this extended logging.

Reply:
This level of detail could be logged at the `trace` level, we could use that instead of having a special flag.

Reply:
I thought that `debug` is the lowest level of logging.